storage/redis/client.go

```diff
@@ -128,9 +127,14 @@ func (client *Client) GetAll(ctx context.Context, keys storage.Keys) (_ storage.
 	}
 
 	results, err := client.db.MGet(keyStrings...).Result()
+	if len(results) == 0 && err.Error() == "ERR wrong number of arguments for 'mget' command" {
+		fmt.Println("No Revocations Present")
```

Comment:
using `fmt` here

Reply:
This redis client isn't specific to revocationDB. Unless I'm misunderstanding something, I don't think anything revocationDB-specific should go here.

Reply:
oh good point

Reply:
I added this so when I run `identity revocations --revocation-dburl="redis://127.0.0.1:6379?db=0" I don't get that mget error message that occurs if there are 0 items the db. Am I missing something?

Reply:
I was not aware of that error. Perhaps it would be better to check for that error in the revocation db package. Also, if the redis package exports that error maybe we can check for it not directly than comparing the error string.